src/pheval/analyse/benchmark_generator.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from typing import Callable
+
+from pheval.analyse.benchmarking_data import AnalysisResults, TrackRunPrioritisation
+from pheval.constants import (
+    DISEASE_PLOT_FILE_PREFIX,
+    DISEASE_PLOT_Y_LABEL,
+    GENE_PLOT_FILE_PREFIX,
+    GENE_PLOT_Y_LABEL,
+    VARIANT_PLOT_FILE_PREFIX,
+    VARIANT_PLOT_Y_LABEL,
+)
+
+
+@dataclass
+class BenchmarkPrioritisationOutputGenerator:
```

Comment:
Rename this to `BenchmarkOutputGenerator`? the 'Prioritisation' bit doesn't make sense to me here.